plugins/powerpc/powerpc_cpu.ml

```diff
@@ -0,0 +1,74 @@
+open Core_kernel.Std
+open Bap.Std
+open Powerpc_rtl
+open Powerpc_dsl
+open Powerpc_utils
+open Powerpc_model
+open Powerpc_types
+
+let size_of_width x =
+  let x = int_of_bitwidth x in
+  match Size.of_int x with
+  | Ok s -> s
+  | Error _ -> ppc_fail "invalid size: %d" x
+
+let find m reg =
+  let (module M : PowerPC) = m in
+  let open M in
+  let open E in
+  let find_reg regs reg = String.Map.find regs (Reg.name reg) in
```

Comment:
use `Map.find` instead of `String.Map.find`. First of all it less verbose, second it confusing, third it make the code less general, and last but not least, it will break in core v0.10. They are removing Maps from the modules.